codegen/src/test/resources/software/amazon/awssdk/codegen/poet/client/test-async-client-class.java

```diff
@@ -1070,6 +1061,21 @@ public void close() {
                                 .exceptionBuilderSupplier(InvalidInputException::builder).httpStatusCode(400).build());
     }
 
+    private static List<MetricPublisher> resolveMetricPublishers(SdkClientConfiguration clientConfiguration,
```

Comment:
Out of curiosity, why do we move this method out of  `MetricUtils`?

Reply:
Discussed this with @millems. `MetricUtils` is `@SdkInternal`, and we'd rather keep it that way than expose it as a protected API.